resview.py

```diff
@@ -702,17 +754,17 @@ def main():
         plotter.open_movie(anim_filename, options.framerate)
         plotter.show(auto_close=False)
 
-        for k in scalar_bars.keys():
-            v = scalar_bars[k]
-            clim = (nm.min([iv[0][0] for iv in v]),
-                    nm.max([iv[0][1] for iv in v]))
-            scalar_bars[k] = (clim, v[0][1])
+        for k in scalar_bar_limits.keys():
+            lims = scalar_bars[k]
```

Comment:
Pyflakes says that `scalar_bars` is undefined here.

Reply:
Of course, Pyflakes is right, there should be `scalar_bars_limits`.

Reply:
I forgot to test the animations.